test/unit/featureprocess.js

```diff
@@ -18,13 +19,13 @@ describe('Layer with Feature process', function () {
     const placement = { coord: new Coordinates('EPSG:4326', 1.5, 43), range: 300000 };
     const viewer = new GlobeView(renderer.domElement, placement, { renderer });
 
-    function extrude() {
-        return 5000;
-    }
+    // function extrude() {
+    //     return 5000;
+    // }
 
-    function color() {
-        return new THREE.Color(0xffcc00);
-    }
+    // function color() {
+    //     return new THREE.Color(0xffcc00);
+    // }
```

Comment:
Could you remove these commented lines ?

Reply:
yes